aidants_connect_web/templates/public_website/statistiques.html

```diff
@@ -15,110 +15,127 @@
+              <td>{{ demarche.title }}</td>
+              <td>{{ demarche.value }}</td>
+            </tr>
+            {% endfor %}
+          </table>
+        </canvas>
       </div>
 
       <br>
 
-      <h2>Usagers</h2>
+      <h2>Déploiement</h2>
       <div class="grid">
         <div class="tile-box">
           <div class="tile tile-grey text-center">
-            <h3 class="font-weight-normal">Usagers</h3>
-            <strong>{{ usagers_with_mandat_count }}</strong>
+            <h3 class="font-weight-normal">Aidants habilités</h3>
+            <strong>{{ aidants_count }}</strong>
           </div>
         </div>
         <div class="tile-box">
           <div class="tile tile-grey text-center">
-            <h3 class="font-weight-normal">Usagers actifs</h3>
-            <strong>{{ usagers_with_active_mandat_count }}</strong>
+            <h3 class="font-weight-normal">Aidants en cours d’habilitation</h3>
+            <strong>{{ aidants_habilitating_count }}</strong>
           </div>
         </div>
-      </div>
-
-      <br>
-
-      <h2>Démarches France Connect accompagnées</h2>
-      <div class="grid">
         <div class="tile-box">
-            <div class="tile tile-grey text-center">
-              <h3 class="font-weight-normal">Depuis le lancement</h3>
-              <strong>
-                Usagers accompagnés : {{ usagers_helped_count }}
-                <br>
-                Connexions effectuées : {{ autorisation_use_count }}
-                <br>
-              </strong>
-            </div>
+          <div class="tile tile-grey text-center">
+            <h3 class="font-weight-normal">Structures habilitées</h3>
+            <strong>{{ organisations_habilitated_count }}</strong>
           </div>
+        </div>
         <div class="tile-box">
           <div class="tile tile-grey text-center">
-            <h3 class="font-weight-normal">30 derniers jours</h3>
-            <strong>
-              Usagers accompagnés : {{ usagers_helped_recent_count }}
-              <br>
-              Connexions effectuées : {{ autorisation_use_recent_count }}
-              <br>
-            </strong>
+            <h3 class="font-weight-normal">Structures en cours d’habilitation</h3>
+            <strong>{{ organisations_not_habilitated_count }}</strong>
           </div>
         </div>
       </div>
-      
-      <h3>Par type de démarche</h3>
-      <div class="grid small-grid">
-        {% for stat_item in demarches_count %}
-          <div class="tile-box">
-            <div class="tile tile-grey text-center">
-              <h4 class="font-weight-normal">
-                <img class="icon-inline" src="{{ stat_item.icon }}" alt="" />
-                {{ stat_item.title }}
-              </h4>
-              <strong>{{ stat_item.value }}</strong>
-            </div>
-          </div>
-        {% endfor %}
-      </div>
+
     </div>
   </div>
 </section>
+
+<script src="https://code.jquery.com/jquery-3.6.0.min.js"></script>
```

Comment:
Est-ce que, plutôt que d'embarquer une version de jQuery pour faire une bête requête JSON, on pourait pas embarquer les données dans le HTML avec [`json_script`](https://docs.djangoproject.com/en/4.0/ref/templates/builtins/#json-script) ? @mrjmad, une objection ? Ça pourrait poser des problèmes de sécurité ?

Reply:
j'allais dire que je savais pas si ca allait passer avec notre content policy mais la doc django indique que : " This is compatible with a strict Content Security Policy that prohibits in-page script execution. It also maintains a clean separation between passive data and executable code. " 

donc je pense que ça doit le faire.

Reply:
tu as raison, en plus ça simplifie bien le code :)